src/components/Sidebar/sidebar.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Sidebar, Menu, MenuItem, useProSidebar } from "react-pro-sidebar";
+import { Link } from "react-router-dom";
+
+function Layout() {
+  const { collapseSidebar } = useProSidebar();
+
+  return (
+    <div style={{ display: "flex", height: "100%" }}>
+      <Sidebar>
+        <Menu>
+          <MenuItem routerLink={<Link to="/reviews" />}>
+            View all reviews
+          </MenuItem>
+          <MenuItem routerLink={<Link to="/calendar" />}> Calendar</MenuItem>
+          <MenuItem routerLink={<Link to="/e-commerce" />}>
+            {" "}
+            E-commerce
+          </MenuItem>
+        </Menu>
+      </Sidebar>
+      <main>
+        <button onClick={() => collapseSidebar()}>Collapse</button>
+      </main>
+    </div>
+  );
+}
+
+export default Layout;
```

Comment:
This collapsing sidebar is really lovely. I'd consider making the text into symbols or bars when it's collapsed for good user experience.